docs/shim-rune.md

```diff
@@ -134,7 +134,7 @@ uses the following methods to run the app enclave container.
 ### Occlum
 
 Refer to the occlum guide of
-[Runtime boot pre-generated UnionFS image](https://github.com/qzheng527/occlum/blob/combine_jsons/demos/runtime_boot).
+[Runtime boot pre-generated UnionFS image](https://github.com/occlum/occlum/tree/master/demos/runtime_boot).
```

Comment:
Please double check this link @qzheng527

Reply:
this is correct but to make it match with the code, we need #180